.ruby-version

```diff
@@ -1 +1 @@
-2.5.0
+3.3.3
```

Comment:
Can we make this just `3.3` so that folks with, for example, `3.3.4` don't need to downgrade?